multiqc/modules/pangolin/pangolin.py

```diff
@@ -200,16 +204,16 @@ def pangolin_table(self):
             "hidden": True,
         }
 
-        headers["pangoLEARN_version"] = {
-            "title": "PangoLEARN version",
-            "description": "The dated version of the pangoLEARN model installed.",
+        headers["scorpio_version"] = {
```

Comment:
I'm not familiar with Pangolin, but could you please comment on changing the tools which versions are reported here? To make sure it's not accidental.

Reply:
Sure - in the code or just here? Pangolin is for assigning lineages to SARS-CoV-2. Over time, the method of performing this has changed, which also impacts various aspects of performance including speed and accuracy. In v4 (the latest), the method changed from [PangoLEARN](https://cov-lineages.org/resources/pangolin/pangolearn.html) to UShER. There is also another tool (scorpio) and database ([constellations](https://cov-lineages.org/constellations.html)), built in to Pangolin. The latest output of Pangolin runs these tools and reports their results. Therefore the version is important. Here are the [full docs](https://cov-lineages.org/resources/pangolin/output.html) for each field in the ouput; you'll notice that `pangoLEARN_version` is no longer reported as it is no longer run by default.

Reply:
Thanks a lot for the explanation! Was curious about removing the previous `pangoLEARN_version` and `pango_version` columns, instead of keeping them along for users of older Pangolin versions. But perhaps we don't need to worry about that, the columns were hidden anyway so unlikely were very popular.

Reply:
I slightly updated the PR title and the description to reflect this change, if you don't mind (this is what I meant about leaving the comment 😅 )